tests/director/tests/formation_templates_test.go

```diff
@@ -332,6 +332,21 @@ func TestModifyFormationTemplateWebhooks(t *testing.T) {
 		assertions.AssertWebhooks(t, []*graphql.WebhookInput{webhookInput}, []graphql.Webhook{*updatedFormationTemplate.Webhooks[0]})
 	})
 
+	urlUpdated := "https://test.com"
+	t.Run("Update formation template webhook", func(t *testing.T) {
```

Comment:
**_minor:_** What do you think for `Update formation template webhook globally`?

Also, there were a TODOs in the formation_api_test.go file related to the update FT webhook globally. I'm not sure if they will be adapter as part of this PR or in a separate one. Just a heads-up :)

Reply:
Nice catch, I didn't realize the other PR was merged. I'll adapt them here.